docs/topics/mta-cli-args.adoc

```diff
@@ -146,39 +172,40 @@ Specify the path to the directory to output the report information generated by
 --output <OUTPUT_REPORT_DIRECTORY>
 ----
 
-* The `--output` argument is mandatory. If omitted, an error is returned:
+* If omitted, the report will be generated in an `<INPUT_ARCHIVE_OR_DIRECTORY>.report` directory.
+* If the output directory exists, you will be prompted with the following (with a default of N).
 +
 [source,options="nowrap",subs="attributes+"]
 ----
-Error: required flag(s) "output" not set 
+Overwrite all contents of "/home/username/<OUTPUT_REPORT_DIRECTORY>" (anything already in the directory will be deleted)? [y,N]
 ----
 
-However, if you specify the `--overwrite` argument, {ProductShortName} will proceed to delete and re-create the directory. See the description of this argument for more information.
+However, if you specify the `--overwrite` argument, {ProductShortName} will proceed to delete and recreate the directory. See the description of this argument for more information.
 
 [id="cli-source-argument_{context}"]
 == Setting the source technology
 
-A space-delimited list of one or more source technologies, servers, platforms, or frameworks to migrate from. This argument, in conjunction with the `--target` argument, helps to determine which rulesets are used. Use the `--list-sources` argument to list all available sources.
+A space-delimited list of one or more source technologies, servers, platforms, or frameworks to migrate from. This argument, in conjunction with the `--target` argument, helps to determine which rulesets are used. Use the `--listSourceTechnologies` argument to list all available sources.
```

Comment:
Our beloved IBM SG suggest to avoid anthropomorphism as much as we can -_- How about:

```suggestion
A space-delimited list of one or more source technologies, servers, platforms, or frameworks to migrate from. You can use this argument, in conjunction with the `--target` argument, to determine which rulesets are used. Use the `--listSourceTechnologies` argument to list all available sources.
```